### PR TITLE
Added support for DAM 14.7 (LTS)

### PR DESCRIPTION
**Pull request by eytannnaim**

### CODE REVIEW

File: modules/aws/mx/hub.tf [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+  hub_action_set = local.dam_version != "14.7" ? local._hub_action_set : local._hub_action_set147
+  hub_action_set_action = local.dam_version != "14.7" ? local._hub_action_set_action : local._hub_action_set_action147
 
-  hub_configuration = var.hub_details == null ? [] : concat([{
+  dam_version_major = split(".", var.dam_version)[0]
+  dam_version_minor = split(".", var.dam_version)[1]
+  dam_version = "${local.dam_version_major}.${local.dam_version_minor}"
+  # Archiving action set is created differently on 14.7
+
+  action_set_item = var.hub_details == [] ? null : local.dam_version != "14.7" ? [{
```
Comment (assaf13) [inline modules/aws/mx/hub.tf]: The condition here is different than hub_configuration = var.hub_details == null ? [] : concat...
Is it ok ?

File: modules/aws/mx/hub.tf [hunk 2]
```diff
@@ -12,13 +21,26 @@ locals {
...
+    url_path = "SecureSphere/api/v1/conf/actionSets/${local.hub_action_set}/${local.hub_action_set}"
+    payload = jsonencode({
+      "type" : "SonarArchiver",
+      "host" : try(var.hub_details.address, null),
+      "port" : try(var.hub_details.port, null),
+      "apiToken" : try(var.hub_details.access_token, null)
+      "strictCertificateChecking" : false
+      "actionInterface": "Send to Sonar"
+      }
+    )
```
Comment (assaf13) [inline modules/aws/mx/hub.tf:40]: Does the only difference at the payload is that it conains actionInterface and missing enabled ?

File: modules/aws/mx/hub.tf (hunk 1 above)
Comment (eytannnaim) [reply modules/aws/mx/hub.tf]: You are absolutely right. Fixing.

File: modules/aws/mx/hub.tf (hunk 2 above)
Comment (eytannnaim) [reply modules/aws/mx/hub.tf:40]: Yes

**Pull request merged**